Shorthand syntax for empty divs and spans

ssahara opened the pull request:
This PR implement shorter syntax `<WRAP|div|block|wrap|span|inline [classname] />` that allow shorthand for empty divs/spans such as `<WRAP clear/>` or  `<span #id/>`.

This request is replacement of #59 that consist with wired commit history.

selfthinker commented on syntax/div.php:
```diff
@@ -101,14 +105,16 @@ function render($mode, Doku_Renderer $renderer, $indata) {
                     $attr = $wrap->buildAttributes($data, 'plugin_wrap');
 
                     $renderer->doc .= '<div'.$attr.'>';
+                    if ($state == DOKU_LEXER_SPECIAL) $renderer->doc .= '</div>';
```
That is not necessary, you're already in the `DOKU_LEXER_SPECIAL` case. (The same is true in the `span.php`.) Is that left over from previous tries?

ssahara replied:
It is necessary because there is no `break` statement before  line 108 `case DOKU_LEXER_SPECIAL`.

selfthinker replied:
Ah, okay.

selfthinker commented:
Yup, this works fine. :+1:

Could you also update [example.txt](https://github.com/selfthinker/dokuwiki_plugin_wrap/blob/master/example.txt)? The new shorthand syntax should be mentioned at the bottom of the "Basic Syntax" section (I guess your `<WRAP clear/>` and `<span #id/>` examples make sense), and the examples for `clear` and `pagebreak` in the "Miscellaneous" section could be updated as well.

ssahara commented:
OK. please let me see what I can help you on example.txt file.

ssahara commented:
Hello @selfthinker 
I have updated the [example.txt](https://github.com/ssahara/dokuwiki_plugin_wrap/blob/shorthand/example.txt) according to your guidance. Also I fixed an inadequate Japanese translation in `ja/lang.php` file, in which my name found as an author!
